### ISSUE TITLE
Linear regression learning rate

### ISSUE THREAD

**Issue by cklaassen** [cklaassen/MachineLearningFromScratch#15]
When running linear regression the learning rate has to be incredibly small.  Right now it is `alpha=.000001`.  This is clearly an issue with my code and should be looked into.

**Comment by cklaassen** [comment]
I made a mistake when reviewing #17.  When testing it appeared that [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html) and my code returned the same result.  This however was not the case as what was happening is the equation would be `nan(x) + nan`, causing it to not show on the graph.  Due to this I made the assumption that the two lines were overlapping which is in fact not the case.  This is now a major bug in master that I will attempt to immediately resolve.
